sdk/textanalytics/azure-ai-textanalytics/azure/ai/textanalytics/_helpers.py

```diff
@@ -0,0 +1,5 @@
+
+def _get_deserialize():
```

Comment:
I think we can move this to the response_handlers file to avoid creating a new file

Reply:
I moved this to response_handlers.py.